unittests/platformagnostictests.py

```diff
@@ -282,7 +282,7 @@ def test_reconfigure_base_options(self):
         self.assertIn('\nMessage: b_ndebug: true\n', out)
         self.assertIn('\nMessage: c_std: c89\n', out)
 
-        out = self.init(testdir, extra_args=['--reconfigure', '-Db_ndebug=if-release', '-Dc_std=c99'])
+        out = self.init(testdir, extra_args=['--reconfigure', '-Db_ndebug=if-release', '-Dsub:b_ndebug=false', '-Dc_std=c99'])
         self.assertIn('\nMessage: b_ndebug: if-release\n', out)
         self.assertIn('\nMessage: c_std: c99\n', out)
```

Comment:
Don't you need to add additional `self.assertIn` cases here?

Reply:
You are right. I fixed the error causing the base option to be not recognised, but now I realise that the subproject option is still not set from the command line...

Reply:
should be fixed now.